### ISSUE TITLE
Enabled recipe list broken in /next

### ISSUE THREAD

**Issue by dsedivec** [factoriolab/factoriolab#894]
**Describe the bug**
The enable/disable recipes list looks like this on https://factoriolab.github.io/next:

![image](https://user-images.githubusercontent.com/95333/206843014-83af9157-ed84-4fcf-85e2-d8cd518d2723.png)

I added a very ugly purple arrow pointing to the thin sliver that "drops down" when you click in that box.  I would say it is the width of my macOS scroll bar.  No content is visible.

**To Reproduce**
1. Open Firefox private browsing session, so I don't have any cookies or saved data (I _assume_)
2. Go to https://factoriolab.github.io/next
3. Click "skip" on the new landing screen
4. Click the "Optional recipes" drop down

**Expected behavior**
List of recipes is presented (and readable :) )

**Screenshots**
See above

**Desktop (please complete the following information):**
 - OS: macOS 11.6.8 Intel
 - Browser: Firefox Developer Edition (but I also tested in whatever presumably recent Chrome I have installed, in a "Guest" tab with no extensions or cache)
 - Version: 108.0b9

**Additional context**
I did a `git bisect`.  Angular (I think—not familiar with it) wouldn't build several of the commits, so the best I can do is:
```
$ git bisect skip
There are only 'skip'ped commits left to test.
The first bad commit could be any of:
0f2a5146274f78d62d869c0198384283f5f4c636
df5d63438bbc3a0444ace09f0cdd60a57b508544
dc660ae04f85f8720d030897d3b3d9b11c003f5e
We cannot bisect more!
```
Maybe this will help!

**Comment by Valoneu** [comment]
have the same problem, I also cant use saved states in /NEXT and i have them in normal one

**Comment by dcbroad3** [comment]
Closed by #896

**Comment by dcbroad3** [comment]
> have the same problem, I also cant use saved states in /NEXT and i have them in normal one

I can't seem to reproduce this issue, please open another issue for this with repro steps
